build_runner/lib/src/generate/build_impl.dart

```diff
@@ -192,7 +197,25 @@ class _SingleBuild {
         _resolvers = buildImpl._resolvers,
         _resourceManager = buildImpl._resourceManager,
         _verbose = buildImpl._verbose,
-        _writer = buildImpl._writer;
+        _writer = buildImpl._writer {
+    hungActionsHeartbeat = new HungActionsHeartbeat(() {
+      final message = new StringBuffer();
+      const numActionsToLog = 5;
```

Comment:
[nit] Effective Dart calls out `numFoo` as a naming pattern to avoid.
https://www.dartlang.org/guides/language/effective-dart/design#prefer-putting-the-most-descriptive-noun-last

I think `maxActionsToLog` would work here, then `additionalActionCount` below

Reply:
That guideline isn't about the word `num` specifially, I guess what it really wants me to do is something like `actionsToLogMax` (the fact that its a maximum is the important part and should come last?).

Updated here and also refactored the `numActionsCompleted` and `numActionsToLog`.

In general this seems like an oddly specific thing to have in effective dart but im happy to conform to it :man_shrugging:.

Reply:
`num` also goes against https://www.dartlang.org/guides/language/effective-dart/design#avoid-abbreviations

😉

Reply:
lol oh ya, the other guideline I disagree with in there lmao :grinning:

Reply:
I would also agree that `num` is more common than `numberOf`, making it an exception to the rule ("Unless the abbreviation is more common than the unabbreviated term,"), even though it is explicitly called out in the do not section haha.

I think whoever wrote this doc just really doesn't like `num*` variables for some reason.